frontend/src/static/js/v_summary.js

```diff
@@ -430,26 +433,21 @@ window.vSummary = {
 
       return null;
     },
-    sortFiltered() {
-      const full = [];
-      if (!this.filterGroupRepos) {
-        full.push([]);
+    updateSortSelection() {
+      if (this.filterGroupSelection === 'groupByAuthors' && this.filterSort === 'displayName') {
+        this.filterSort = 'searchPath';
+      } else if (this.filterGroupSelection === 'groupByRepos' && this.filterSort === 'searchPath') {
+        this.filterSort = 'displayName';
       }
-
-      this.filtered.forEach((users) => {
-        if (this.filterGroupRepos) {
-          users.sort(window.comparator((ele) => ele[this.filterSort]));
-          full.push(users);
-        } else {
-          users.forEach((user) => full[0].push(user));
-        }
-      });
-
-      if (!this.filterGroupRepos) {
-        full[0].sort(window.comparator((ele) => {
-          const field = ele[this.filterSort];
-          return field.toLowerCase ? field.toLowerCase() : field;
-        }));
+    },
+    sortFiltered() {
+      let full = [];
+      if (this.filterGroupSelection === 'groupByNone') {
+        full[0] = this.groupByNone(this.filtered);
```

Comment:
comment that this is making everything "under one group", not apparent on first glance